modules/ROOT/pages/getting-started-proxy.adoc

```diff
@@ -1,43 +1,47 @@
-= Quick start: Manage an API and set up a proxy for it
+= Getting Started with Managing an API 
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In this walkthrough, we are showing how to create an API that can be managed through API Manager and set up a proxy for it that
-runs in CloudHub.
+You can manage your API from API Manager to apply policies, add contracts, alerts, and SLA tiers. Additionally, you can configure a proxy for your API that runs in CloudHub. You can manage your API using the following steps:
 
-== Creating an API asset in Exchange
+. <<create_an_api, Create an API in Exchange.>>
+. <<import-api-to-API-Manager, Import your API to API Manager.>>
+. <<configure_and_deploy_to_cloudhub, Configure a proxy for your API and deploy to CloudHub.>>
 
-In order to have an API that can be managed through API Manager, the first step consists in creating an API asset in Anypoint Exchange.
+[[create_an_api]]
+== Create an API Asset in Exchange
 
-. Go to Anypoint Exchange https://anypoint.mulesoft.com/exchange
+To manage an API in API Manager, you must first create your API asset in Anypoint Exchange.
+
+To create an API in Exchange:
+
+. Log into Anypoint Platform and select Anypoint Exchange.
 . Click in `New Asset` button
 +
 image::getting-started-proxy-new-asset-button.png[]
 +
-. In `Publish New Asset` view, choose the asset type and an asset name. In case the asset type is RAML, OAS or WSDL, the
-corresponding associated file has to also be specified.
+. In the `Publish New Asset` view, choose the asset type and an asset name. 
++
+For RAML, OAS, or WSDL asset types, you must also provide the corresponding associated file.
 +
 image::getting-started-proxy-new-asset-view.png[width=66%]
 +
-. Click on `Publish button`
+. Click `Publish button`.
```

Comment:
Should it be `Publish` button instead?

Reply:
Changed the formatting to Bold. Done! Thanks!